src/hydrusvideodeduplicator/dedup.py

```diff
@@ -372,13 +371,18 @@ def clear_trashed_files_from_db(self) -> None:
             CHUNK_SIZE = 32
             delete_count = 0
             with SqliteDict(str(DEDUP_DATABASE_FILE), tablename="videos", flag="c") as hashdb:
-                for batched_keys in self.batched(hashdb, CHUNK_SIZE):
-                    is_trashed_result = self.is_files_trashed_hydrus(batched_keys)
-                    for result in is_trashed_result.items():
-                        if result[1] is True:
-                            del hashdb[result[0]]
-                            delete_count += 1
-                    hashdb.commit()
+                total = len(hashdb)
```

Comment:
Maybe accessing database and creating if it does not exist should be moved to a function to avoid rewriting the same thing in every function that accesses the database. With a verbosity parameter of that function for printing it could just be set on for whatever the first function is that accesses the database for the CLI program. Not important if you don't want to do this for this PR though.

Also, probably should return if the length is 0.

You have to use flag="c" which creates a table and database if it does not exist, so it could have no entries. flag="w" is misleading (to me); it actually overwrites the entire table.

from sqlitedict: `'w': open for r/w, but drop tablename contents first (start with empty table)`

Reply:
@ianwal Returning if len()=0 implemented!

I could def see the value of making a function like that, but that's probably better folded into a different PR I think (or is at least more than I want to do for this little PR)